xml/System.Buffers/SequenceReader`1.xml

```diff
@@ -29,8 +29,8 @@
     </Attribute>
   </Attributes>
   <Docs>
-    <typeparam name="T">To be added.</typeparam>
-    <summary>To be added.</summary>
+    <typeparam name="T">The type of the read-only sequence.</typeparam>
+    <summary>Provides methods for reading binary and text data out of a <see cref="T:System.Buffers.ReadOnlySequence{`0}" /> with a focus on performance and minimal or zero heap allocations.</summary>
```

Comment:
@mairaw @rpetrusha the new build threw a warning in this line. It seems the DocId here should be:

```suggestion
    <summary>Provides methods for reading binary and text data out of a <see cref="T:System.Buffers.ReadOnlySequence`1" /> with a focus on performance and minimal or zero heap allocations.</summary>
```

I will commit it, but let me know if you don't agree so we can change it.